src/modm/platform/gpio/sam/pin.hpp.in

```diff
@@ -113,6 +113,10 @@ struct PinMuxMixin<PinConfig, PinConfigs...>
 		{
 			PIOB->PIO_ABCDSR[0] = (PIOB->PIO_ABCDSR[0] & ~(1<<PinConfig::pin)) | (bit0 << PinConfig::pin);
 			PIOB->PIO_ABCDSR[1] = (PIOB->PIO_ABCDSR[1] & ~(1<<PinConfig::pin)) | (bit1 << PinConfig::pin);
+		} else if constexpr (PinConfig::port == PortName::D)
+		{
+			PIOD->PIO_ABCDSR[0] = (PIOD->PIO_ABCDSR[0] & ~(1<<PinConfig::pin)) | (bit0 << PinConfig::pin);
+			PIOD->PIO_ABCDSR[1] = (PIOD->PIO_ABCDSR[1] & ~(1<<PinConfig::pin)) | (bit1 << PinConfig::pin);
```

Comment:
I might be misreading bit0/bit1 macros, but please help me understand:
- isn't `(ABCDSR[0] & ~(1 << pin)) | (bit0 << pin)` effectively the same as `ABCDSR[0] | (1 << pin)`?
- isn't the intent of the second line to `~(0b10 << pin)` instead of `(1 << pin)`? the bits manipulated on each side of the `|` operator are different

Reply:
Whoa, I thought it's `modm::Bit0` and `modm::Bit1`. Definitely a confusing name.

After applying templates I went way over the 100 column limit. Should I try to squeeze this?

Reply:
Oh. I didn't even realize mom::Bit0/Bit1 was a thing. You'll hear no objection from me if you want to rename those local variables :).

Reply:
> After applying templates I went way over the 100 column limit. Should I try to squeeze this?

No, the coding style only refers to the generated source code, the templates don't count.